src/app.tsx

```diff
@@ -180,42 +181,46 @@ export default class App extends React.Component<{}, IState> {
 
     public render() {
         if (this.state.appReady) {
+            const bgRequest = !!document.location.hash;
```

Comment:
I think we should check this for mobile, it returns undefined for `document.location.hash`. 
Also because of this the tests are crashing and the mobile app is crashing, cannot find `document`.

Reply:
If all good, I have updated it to:

```
const bgRequest = Platform.OS === 'web' && !!document.location.hash;
```